pkg/portal/portal.go

```diff
@@ -455,7 +436,23 @@ func (p *portal) getSubscriptionDocument(ctx context.Context, key string) (*api.
 	return doc, err
 }
 
+func (p *portal) getResourceID(r *http.Request) string {
+	apiVars := mux.Vars(r)
+	subscriptionID := apiVars["subscription"]
+	resourceGroup := apiVars["resourceGroup"]
+	clusterName := apiVars["clusterName"]
+	return strings.ToLower(
+		fmt.Sprintf(
+			"/subscriptions/%s/resourceGroups/%s/providers/Microsoft.RedHatOpenShift/openShiftClusters/%s",
+			subscriptionID, resourceGroup, clusterName))
+}
+
 func (p *portal) internalServerError(w http.ResponseWriter, err error) {
 	p.log.Warn(err)
 	http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 }
+
+func (p *portal) badRequest(w http.ResponseWriter, err error) {
+	p.log.Warn(err)
```

Comment:
I don't think we need to use `Warn` here. I expect `Debug` would be more appropriate because we don't really care about bad requests from clients.